pages/attendees.mdx

```diff
@@ -40,8 +40,36 @@ Nearby Subway Stations: <span class="subway subway-u2">U2</span> Messe-Pater, <s
 - Advanced ReasonReact Workshop: [TC.0.58 TC Hall](https://campus.wu.ac.at/en/?campus=1&startstr=Underground%20station%20Messe%20Prater&endstr=TC.0.58%20TC%20Hall&type=0)
 - OCaml Workshop: [TC.3.07 Room](https://campus.wu.ac.at/en/?campus=1&startstr=Underground%20station%20Messe%20Prater&endstr=TC.3.07&type=0)
 
+
+### Journey to the Venue
+
+**FROM:** Airport Vienna (train station) /  **TO:** [Messe-Prater](https://goo.gl/maps/bYq8Ewidw3q) <span class="subway subway-u2">U2</span>
+
+**Note:** Try to avoid CAT trains, since they are 5 times more expensive and are only 5-8 min faster. It’s also a private company, tickets won’t be accepted by the public train service (= ÖBB) and vice versa.
+- Go to the train station located at the airport
+
+- Find a red ÖBB ticket machine or use the ÖBB app
+([iOS](https://itunes.apple.com/at/app/%C3%B6bb/id1041401604?l=en&mt=8) /
+[Android](https://play.google.com/store/apps/details?id=at.oebb.ts&hl=en)) for the ticket
+
+- Buy a ticket to **Wien Messe-Prater** (4,2 EUR)
+- Check the monitors, the ÖBB app or follow this [Maps
+link](https://www.google.at/maps/dir/Vienna+International+Airport+(VIE),+Wien-Flughafen,+1300+Schwechat/Wien+Hbf.,+Vienna/@48.1510338,16.4352152,13z/data=!3m1!4b1!4m19!4m18!1m5!1m1!1s0x476c55ab471abe9b:0x247a52108dd29b4b!2m2!1d16.5665751!2d48.1158333!1m5!1m1!1s0x476da9d6eb3d6139:0x337179155e1ad330!2m2!1d16.3746088!2d48.1859545!2m4!4e2!5e1!5e2!5e3!3e3)
+to find a train to **Wien Hbf (Central Station)**
+- From Wien Hbf, take a <span class="subway subway-u1"> U1 </span> metro (direction Leopoldau) to **Wien Praterstern Bf** (6 stops)
+- From Wien Praterstern, find the <span class="subway subway-u2">U2</span> metro (direction Wien Aspernstraße) to **Messe-Prater**
+(1 stop)
+- From Messe-Prater it's a [7 min walk](https://www.google.at/maps/dir/Messe-Prater+U,+Vienna/Geb%C3%A4ude+LC,+Welthandelsplatz,+Vienna/@48.2154573,16.4038268,17z/data=!3m1!4b1!4m19!4m18!1m5!1m1!1s0x476d071a47b94951:0xabb841058d5d63d6!2m2!1d16.4037671!2d48.2176816!1m5!1m1!1s0x476d0717b8f5b68f:0x6d67c807fd9ab214!2m2!1d16.4084665!2d48.2134275!2m4!4e2!5e1!5e2!5e3!3e3) to the
+conference venue (LC building)
+
 ## After Party
 
-The party will likely take place nearby the conference venue. We will
-announce it before the conference.
+The party will take place at [Kolariks
+Luftburg](https://goo.gl/maps/Pnh2tjNJgtC2) and is close to the venue ([7 min
+walk](https://www.google.at/maps/dir/''/Kolariks+Luftburg,+Prater+128,+1020+Wien/@48.2132187,16.405072,17z/data=!4m14!4m13!1m5!1m1!1s0x0:0xf856f023c1e031c4!2m2!1d16.4089236!2d48.2137796!1m5!1m1!1s0x476d0716d5e2df85:0x5f2720405b0a6139!2m2!1d16.4035807!2d48.2132759!3e3)).
+We will leave as a group shortly after the closer on Day 1, feel free to join us!
```

Comment:
Is it Day 1 or 2? … I think it would be best to mention `on Friday` or `on Friday (12th April)`